lib/Applify.pm

```diff
@@ -149,6 +154,19 @@ OPTION:
 
   print "Usage:\n";
 
+  if ($self->_has_subcommands){
```

Comment:
Just do `%{$self->{subcommands}}` instead.

Reply:
TIL - really, does that work?

Reply:
Yeah 😃